resource/interface/client/radial.lua

```diff
@@ -182,6 +182,15 @@ function lib.removeRadialItem(id)
     refreshRadial(id)
 end
 
+---Removes all items from the global radial menu.
+function lib.clearAllRadialItems()
+    if isOpen then
+        lib.hideRadial()
+    end
```

Comment:
Why not do a check for `not currentRadial` as well and update the menu instead of closing it?